RELEASING.md

```diff
@@ -2,7 +2,7 @@
 
 ## Main Release
 
-Released quarterly on the first day of January, April, July, October.
+Released on January 2nd, April 1st, July 1st and October 15th.
```

Comment:
I'd suggest to keep "quarterly", as even if not bang on, it is still four times per year, and in the same months.

```suggestion
Released quarterly on January 2nd, April 1st, July 1st and October 15th.
```